client/extensions/woocommerce/app/settings/email/email-settings/index.js

```diff
@@ -1,23 +1,224 @@
 	render() {
+		const { loading, loaded, settings } = this.props;
+		const waiting = ! loaded || loading;
+
 		return (
 			<div className="email-settings__container">
+				<div className="email-settings__origin">
+					<Card className="email-settings__origin-title">
+						Origin
+					</Card>
+					<Card className="email-settings__origin-settings">
+						{ waiting && <p className="email-settings__loading" /> }
+						{ loaded && <NotificationsOrigin
+								item={ fromName }
+								recipient={ settings.email[ fromName.field ] }
+								onChange={ this.recipientsChange }
+							/> }
+						{ waiting && <p className="email-settings__loading" /> }
+						{ loaded && <NotificationsOrigin
+								item={ fromAddress }
+								recipient={ settings.email[ fromAddress.field ] }
+								onChange={ this.recipientsChange }
+							/> }
+					</Card>
+				</div>
+				<div className="email-settings__internal-notifications">
+					<Card className="email-settings__internal-notifications-title">
+						<div className="email-settings__section-title-text">
+							{ translate( 'Internal notifications' ) }
+						</div>
+						<div className="email-settings__section-subtitle-text">
+							{ translate( 'Email notifications sent to store staff.' ) }
+						</div>
+					</Card>
+					<Card className="email-settings__internal-notifications-legend">
+						<span className="email-settings__internal-notifications-legend-option">
+							Email
+						</span>
+						<span className="email-settings__internal-notifications-legend-input">
+							Recipients (comma separated)
+						</span>
+						<span className="email-settings__internal-notifications-legend-toggle">
+							Enabled
+						</span>
+					</Card>
+					{
+						waiting &&
+							<Card className="email-settings__loading-card" >
+								{ internalNotifications.map( ( item, index ) => {
+									return <p key={ index } className="email-settings__loading" />;
+								} ) }
+							</Card>
+					}
+					{
+						loaded &&
+							<Card className="email-settings__internal-notifications-settings">
+								{ internalNotifications.map( ( item, index ) => {
+									return <InternalNotification
+										key={ index }
+										item={ item }
+										checked={ 'yes' === settings[ item.field ].enabled }
+										recipient={ settings[ item.field ].recipient }
+										onToggle={ this.notificationsToggle }
+										onChange={ this.recipientsChange }
+									/>;
+								} ) }
+							</Card>
+					}
+				</div>
+				<div className="email-settings__customer-notifications">
+					<Card className="email-settings__customer-notifications-title">
+					<div className="email-settings__section-title-text">
+							{ translate( 'Customer notifications' ) }
+						</div>
+						<div className="email-settings__section-subtitle-text">
+							{ translate( 'Email notifications sent to your customers.' ) }
+						</div>
+					</Card>
+					<Card className="email-settings__customer-notifications-legend">
```

Comment:
This isn't consistent with other settings legend headers, both in markup and in style. See https://github.com/Automattic/wp-calypso/blob/5f15f3365213253724ca0d8798c6cb8160f00bc9/client/extensions/woocommerce/app/settings/shipping/shipping-zone-list.js#L70 for example.